index.html

```diff
@@ -2725,6 +2725,38 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *minhnghia2k3* Contributor card END ________  -->
 
+        <!-- ________ *Manideep* Contributor card START ________  -->
+
+        <div class="card">
+          <p class="name">Manideep byri</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://www.twitter.com/ManideepByri" target="_blank">@ManideepByri</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/Manideep-2" target="_blank">@Manideep-2</a>
+          </p>
+          <p class="about">
+            I'm Manideep, a Computer Science student. Passionate about coding, I explore the ever-evolving tech landscape.
+            Movies and anime are my escape, providing inspiration and balance.
+            Eager to merge creativity and logic as I navigate the exciting journey of earning my Bachelor's in Technology.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">http://github.com/git-tips/tips</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">https://alison.com/</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">https://grow.google/intl/uk/</a>
```

Comment:
Hi there, just a minor problem before we merge (don't worry, this is a normal part of the review process).

Please note that all three links (e.g "https://github.com/git-tips/tips") should go into their respective href section. The places where your links currently are should be the name of the resource.
```suggestion
                <a href="http://github.com/git-tips/tips" target="_blank" title="First Resource">GitHub Tips</a>
              </li>
              <li>
                <a href="#" target="_blank" title="Second Resource">https://alison.com/</a>
              </li>
              <li>
                <a href="#" target="_blank" title="Third resource">https://grow.google/intl/uk/</a>
```

It is also recommended that you use https instead of http for your github link.
```suggestion
                <a href="**https**://github.com/git-tips/tips" target="_blank" title="First Resource">GitHub Tips</a>
              </li>
              <li>
                <a href="#" target="_blank" title="Second Resource">https://alison.com/</a>
              </li>
              <li>
                <a href="#" target="_blank" title="Third resource">https://grow.google/intl/uk/</a>
```

Please make these changes and make another pull request,